source/administration/command-line-tools.rst

```diff
@@ -880,6 +882,43 @@ platform config validate
 
         sudo ./platform config validate
 
+platform sampledata
+-------------------
+
+  Description
+    .. versionadded:: 4.7
+      Generate sample data and populate the Mattermost database.
+
+  Format
+    .. code-block:: none
+
+      platform sampledata
+
+  Example
+    .. code-block:: none
+
+      sudo ./platform sampledata --seed 10 --teams 4 --users 30
+
+  Options
+    .. code-block:: none
+
+          -u, --users int                      The number of sample users. (default 15)
+              --profile-images string          Optional. Path to folder with images to randomly pick as user profile image.
+          -t, --teams int                      The number of sample teams. (default 2)
+              --team-memberships int           The number of sample team memberships per user. (default 2)
+              --channels-per-team int          The number of sample channels per team. (default 10)
+              --channel-memberships int        The number of sample channel memberships per user in a team. (default 5)
+              --posts-per-channel int          The number of sample post per channel. (default 100)
+              --direct-channels int            The number of sample direct message channels per user. (default 30)
+              --group-channels int             The number of sample group message channels per user. (default 15)
+              --posts-per-direct-channel int   The number of sample posts per direct message channel. (default 15)
+              --posts-per-group-channel int    The number of sample post per group message channel. (default 30)
+          -s, --seed int                       Seed used for generating the random data. (default 1)
```

Comment:
@jespino can the seed be clarified a bit? Not sure I totally understand from what's written here

Reply:
I think, now is clearer but I can try to rewrite all the sentence. Maybe we can change the whole text to something like "Change the seed to generate a different set of data". It is a cli command for help developers whose normally are familiarized with the "seed" concept.